test/ctia/http/routes/common_test.clj

```diff
@@ -34,6 +35,28 @@
               :lt to}
              (sut/coerce-date-range from to))))))
 
+(deftest full-text-search-schema-functions
```

Comment:
your `testing` description are the name of each function. 
Please, make a `deftest` per function.